autofill/autofill-api/src/main/res/layout/activity_autofill_settings.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Why do we have resources in the api module? Same goes for the strings.

Reply:
They had already been copied to the `impl` but this hadn't been tidied up. Gone now.